addons/account/models/chart_template.py

```diff
@@ -1002,3 +1020,27 @@ def _load_translations(self, langs=None, companies=None):
                                     xml_id = f"account.{company.id}_{_xml_id}"
                                     translation_importer.model_translations[mname][fname][xml_id][lang] = value
         translation_importer.save(overwrite=False)
+
+    def _load_code_translations(self, accounts, langs=None):
```

Comment:
Is there a reason we put langs as a parameter but don't use it ? i see that the function "_load_translations" and "_load_code_translations" in transifex_code_translation.py did the same.

Also is this function an overide of the one from transifex_code_translation ? in that case it may be missing a super ? or just rename the function a bit :smile:

Reply:
I renamed the function to `_load_account_name_code_translations` to avoid any ambiguity.

I used the `langs` parameter since it was used for `_load_translation`. But since it is not used anywhere I removed it.